src/main/java/com/game/racing/domain/car/Car.java

```diff
@@ -0,0 +1,39 @@
+package com.game.racing.domain.car;
+
+import com.game.racing.domain.generator.NumberGenerator;
+import com.game.racing.domain.position.Position;
+import com.game.racing.view.ResultView;
+
+public class Car {
+
+    public static final Integer MIN_VALUE_TO_MOVE_CAR = 4;
+    private final Position position;
+
+    private final NumberGenerator numberGenerator;
```

Comment:
상수와 필드는 개행으로 분리해두면, 가독성이 더 올라갈듯 합니다.

```suggestion
    public static final Integer MIN_VALUE_TO_MOVE_CAR = 4;
    
    private final Position position;
    private final NumberGenerator numberGenerator;
```

Reply:
좋은 의견 감사합니다. 해당 내용 반영해보았습니다!